deephub/variants/definition.py

```diff
@@ -182,13 +207,17 @@ def train(self, trainer: Trainer) -> ModelBase:
             eval_feeder = self.create_feeder('train.eval_feeder')
         else:
             eval_feeder = None
-
+        if self.has('train.exporters'):
+            exporters = self.create_exporters('train.exporters')
+        else:
+            exporters = None
         # Dump model and train params from variants yaml
         self._dump_variant_definition()
 
         # Invoke training
```

Comment:
Do we really need the sub_get method? Why don't we just simply add to the get method the exclude_keys param?